pkg/detectors/jdbc/jdbc.go

```diff
@@ -15,7 +18,7 @@ type Scanner struct{}
 var _ detectors.Detector = (*Scanner)(nil)
 
 var (
-	keyPat = regexp.MustCompile(`(?i)jdbc:[\w]{3,10}:\/\/\w[\s\S]{0,512}?password[=: \"']+(?P<pass>[^<{($]*?)[ \s'\"]+`)
+	keyPat = regexp.MustCompile(`(?i)jdbc:[^\s"']+`)
```

Comment:
This might be a bit too permissive a pattern for unverified results.. I'll revisit this.